plugins/action/zos_copy.py

```diff
@@ -124,9 +125,6 @@ def run(self, tmp=None, task_vars=None):
         #     msg = "Invalid port provided for SFTP. Expected an integer between 0 to 65535."
         #     return self._exit_action(result, msg, failed=True)
 
-        if (not force) and self._dest_exists(src, dest, task_vars):
```

Comment:
I went through this file, but don't see where this functionality is replaced... shouldn't we kick an error on force=False and destination exists?  Or am I overlooking something?

Reply:
Because the module now tries to use empty data sets when it's appropriate, the module (instead of the action plugin) checks whether we should throw an error when the destination exists and `force=False` on line 1987.

```diff
+    if not does_destination_allow_copy(
+       src,
+        is_src_dir,
+        dest_name,
+        dest_exists,
+        dest_member_exists,
+        dest_ds_type,
+        is_uss,
+        force,
+        volume
+    ):
+        module.fail_json(msg="{0} already exists on the system, unable to overwrite unless force=True is specified.".format(dest))
```